src/main/java/com/github/valfirst/slf4jtest/TestLoggerAssert.java

```diff
@@ -46,16 +37,18 @@ public TestLoggerAssert hasLogged(Level level, String message, Object... argumen
     */
     public TestLoggerAssert hasLogged(
```

Comment:
Did we want to mark this as deprecated?

Reply:
I would say "yes", if it was some business code. 
But as for the utility library I prefer to keep these methods non-deprecated: `LoggingEvent`-based methods provide more flexibility, but at the same time non-`LoggingEvent`-based methods gives simplicity for users who don't want to dive deep